Translation

daniel-teddy opened the pull request:
@kirakoki review needed before merge
added product pages
checked responsiveness
added quote page

daniel-teddy committed:
wip

daniel-teddy committed:
wip - product page

product page completed
-feeding data to array
-adding necessary ids to display the right datas

daniel-teddy committed:
wip

added files for products pages

daniel-teddy committed:
wip

finished fitness product page for each product

daniel-teddy committed:
wip

done
-fitness devices
-cryotherapy devices

daniel-teddy committed:
wip

added all products pages
linked products to their pages
# todos:
- checking responsivness of each product page
- add quote page

daniel-teddy committed:
wip

-added quote page
todo:
- quote page functionality

daniel-teddy committed:
wip

functionnal quote page
# todo:
-add api to receive submitted form

daniel-teddy committed:
wip

added links from products to quote page

daniel-teddy committed:
work in progress

small fixes

AhmetMustafa reviewed (commented):
Manually reviewed, some inconsistencies pointed out for Daniel to fix, in general looking good both on Desktop and Mobile.

daniel-teddy committed:
minor changes

- text animation fixed
- product responsiveness fixed
- translation text edited for main page
-

daniel-teddy committed:
wip - fix

fixed minor issues
checked furniture products

daniel-teddy commented:
fix ready to merge
